package.json

```diff
@@ -29,10 +29,10 @@
     "test": "vitest run",
     "clean:snapshots": "rimraf cypress/snapshots",
     "cy:base-and-regression": "npm run cy:base && npm run cy:regression",
-    "cy:open:base": "visual_regression_log=debug cypress open --e2e -b chrome -C cypress.base.config.ts",
-    "cy:open:regression": "visual_regression_log=debug cypress open --e2e -b chrome -C cypress.regression.config.ts",
-    "cy:base": "visual_regression_log=debug cypress run -C cypress.base.config.ts",
-    "cy:regression": "visual_regression_log=debug cypress run -C cypress.regression.config.ts"
+    "cy:open:base": "set visual_regression_log=debug && cypress open --e2e -b chrome -C cypress.base.config.ts",
+    "cy:open:regression": "set visual_regression_log=debug && cypress open --e2e -b chrome -C cypress.regression.config.ts",
+    "cy:base": "set visual_regression_log=debug && cypress run -C cypress.base.config.ts",
+    "cy:regression": "set visual_regression_log=debug && cypress run -C cypress.regression.config.ts"
```

Comment:
This config won't work on Mac.

The good news is that we can use  [cross-env](https://www.npmjs.com/package/cross-env) to solve the problem.

Can you try that approach?   🙏